ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/application_traffic_recognition.schema.yml

```diff
@@ -155,6 +155,77 @@ keys:
 
                     When the `protocols` list contain both `tcp` and `udp`, this key value
                     must be the same as `udp_dest_port_set_name`.
+          l4_applications:
```

Comment:
Given that we are repeating a lot of things it may be worth doing a `def:` to reuse in this scenario

Reply:
Latest push should take care of this @gmuloc